config/locales/client.en.yml

```diff
@@ -121,6 +121,9 @@ en:
 
       composer:
         toggle_toolbar: "Toggle toolbar"
+        italic_text: "emphasized text"
+        bold_text: "strong text"
+        code_text: "indent preformatted text by 4 spaces"
```

Comment:
Correct me if I'm wrong 😉 but it doesn't indent but surrounds with \`\`? 🙂

Reply:
hmm good point...will just change to `code text`

Reply:
This only shows up if you press the shortcut with nothing selected anyway